docs/config_template.yaml

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+################################
+#                              #
+#    EXAMPLE CONFIGURATION     #
+#                              #
+################################
+---
+# The common config for kperf
+kperf:
+  # TODO: The log_level of kperf output
+  log_level: info
+
+  # The directory kperf will save output to
+  # Default is current working directory.
+  output: "."
+
+# The config for Knative Service Provision load test.
+svc_provision:
+  # The total number of Knative Service to generate.
+  number: 100
+
+  # NOTE: either use namespace or namespace_prefix & namespace_range.
+  # When namespace is not empty, Knative Services will be created in the namespace.
+  namespace: "default"
+
+  # The Knative Services will be created and distributed in the namespaces with the prefix and range.
+  namespace_prefix: ""
+  namespace_range: ""
```

Comment:
What’s the namespace range?

Reply:
We want to support to generate Knative Services distributed in multiple namespaces like ns-1, ns-2,...ns-20, so range is for the index